tests/pytorch/test_pytorch_model_export.py

```diff
@@ -900,7 +900,9 @@ def forward(self, *args, **kwargs):  # pylint: disable=arguments-differ
     scores = PredictionsResponse.from_json(resp.content.decode("utf-8")).get_predictions(
         predictions_format="ndarray"
     )
-    assert_array_almost_equal(scores, model(input_ids).detach().numpy(), rtol=1e-6)
+    assert_array_almost_equal(
+        scores, model(input_ids).detach().numpy(), rtol=1e-6  # pylint: disable=not-callable
```

Comment:
`E1102 model is not callable (cot-callable)` error has occurred here. I`m still investigating.

Reply:
Did you get this error on CI?

Reply:
No, I run `./dev/lint.sh` and pre-commits.

Reply:
What is your pylint version? On CI, 2.14.4 is used.

Reply:
This sounds like a false positive because `model` is callable.

Reply:
![image](https://user-images.githubusercontent.com/61897166/206836061-9ca74552-1cb2-4979-99e1-8278f2b7cad2.png)

Reply:
Interesting. Is `transformers` installed?

Reply:
Yes
![image](https://user-images.githubusercontent.com/61897166/206836220-f5cc71ad-a02b-434d-b9df-423e3c73fd4d.png)

Reply:
Can you share the output of `pip list`?

Reply:
```
Package                       Version     Editable project location
----------------------------- ----------- ----------------------------
adal                          1.2.7
aiofiles                      22.1.0
aiohttp                       3.8.3
aiokafka                      0.8.0
aiosignal                     1.3.1
alabaster                     0.7.12
alembic                       1.8.1
anyio                         3.6.2
argcomplete                   2.0.0
astroid                       2.11.7
asttokens                     2.2.0
async-timeout                 4.0.2
attrs                         22.1.0
azure-common                  1.1.28
azure-core                    1.26.1
azure-graphrbac               0.61.1
azure-identity                1.12.0
azure-mgmt-authorization      2.0.0
azure-mgmt-containerregistry  10.0.0
azure-mgmt-core               1.3.2
azure-mgmt-keyvault           10.1.0
azure-mgmt-resource           21.2.1
azure-mgmt-storage            20.1.0
azure-storage-blob            12.14.1
azureml-core                  1.47.0
Babel                         2.11.0
backcall                      0.2.0
backports.tempfile            1.0
backports.weakref             1.0.post1
bcrypt                        4.0.1
black                         22.3.0
boto3                         1.26.22
botocore                      1.29.22
Brotli                        1.0.9
cachetools                    5.2.0
certifi                       2022.9.24
cffi                          1.15.1
cfgv                          3.3.1
charset-normalizer            2.1.1
click                         8.1.3
cloudpickle                   2.2.0
colorama                      0.4.6
commonmark                    0.9.1
contextlib2                   21.6.0
contourpy                     1.0.6
coverage                      6.5.0
cryptography                  38.0.4
cycler                        0.11.0
databricks-cli                0.17.4.dev0
decorator                     5.1.1
dill                          0.3.6
distlib                       0.3.6
docker                        6.0.1
docutils                      0.16
entrypoints                   0.4
exceptiongroup                1.0.4
executing                     1.2.0
fastapi                       0.88.0
filelock                      3.8.0
FLAML                         1.0.14
Flask                         2.2.2
fonttools                     4.38.0
frozenlist                    1.3.3
future                        0.18.2
gevent                        22.10.2
geventhttpclient              2.0.2
gitdb                         4.0.10
GitPython                     3.1.29
google-api-core               2.11.0
google-auth                   2.15.0
google-cloud-core             2.3.2
google-cloud-storage          2.6.0
google-crc32c                 1.5.0
google-resumable-media        2.4.0
googleapis-common-protos      1.57.0
greenlet                      2.0.1
grpcio                        1.51.1
gunicorn                      20.1.0
h11                           0.14.0
huggingface-hub               0.11.1
humanfriendly                 10.0
hyperopt                      0.2.7
identify                      2.5.9
idna                          3.4
imagesize                     1.4.1
importlib-metadata            5.1.0
importlib-resources           5.10.0
iniconfig                     1.1.1
ipython                       8.7.0
isodate                       0.6.1
isort                         5.10.1
itsdangerous                  2.1.2
jedi                          0.18.2
jeepney                       0.8.0
Jinja2                        3.0.3
jmespath                      1.0.1
joblib                        1.2.0
jsonpickle                    2.2.0
kafka-python                  2.0.2
kaleido                       0.2.1
kiwisolver                    1.4.4
knack                         0.10.1
kubernetes                    25.3.0
lazy-object-proxy             1.8.0
lightgbm                      3.3.3
livereload                    2.6.3
llvmlite                      0.39.1
Mako                          1.2.4
Markdown                      3.4.1
MarkupSafe                    2.1.1
matplotlib                    3.6.2
matplotlib-inline             0.1.6
mccabe                        0.7.0
mlflow                        2.0.2.dev0  /home/akira/akira/oss/mlflow
mlserver                      1.2.0
mlserver-mlflow               1.2.0
moto                          4.0.11
msal                          1.20.0
msal-extensions               1.0.0
msrest                        0.7.1
msrestazure                   0.6.4
multidict                     6.0.3
mypy-extensions               0.4.3
ndg-httpsclient               0.5.1
networkx                      2.8.8
nodeenv                       1.7.0
numba                         0.56.4
numpy                         1.23.5
oauthlib                      3.2.2
orjson                        3.8.3
packaging                     21.3
pandas                        1.5.2
paramiko                      2.12.0
parso                         0.8.3
pathspec                      0.10.2
pexpect                       4.8.0
pickleshare                   0.7.5
Pillow                        9.3.0
pip                           22.3.1
pkginfo                       1.9.2
platformdirs                  2.5.4
plotly                        5.11.0
pluggy                        1.0.0
portalocker                   2.6.0
pre-commit                    2.20.0
prometheus-client             0.15.0
prometheus-flask-exporter     0.21.0
prompt-toolkit                3.0.33
protobuf                      4.21.10
ptyprocess                    0.7.0
pure-eval                     0.2.2
py-grpc-prometheus            0.7.0
py4j                          0.10.9.5
pyarrow                       10.0.1
pyasn1                        0.4.8
pyasn1-modules                0.2.8
pycparser                     2.21
pydantic                      1.10.2
Pygments                      2.13.0
PyJWT                         2.6.0
pylint                        2.14.4
PyNaCl                        1.5.0
pyOpenSSL                     22.1.0
pyparsing                     3.0.9
pysftp                        0.2.9
PySocks                       1.7.1
pyspark                       3.3.1
pytest                        7.2.0
pytest-cov                    4.0.0
pytest-localserver            0.5.0
python-dateutil               2.8.2
python-dotenv                 0.21.0
python-rapidjson              1.9
pytz                          2022.6
PyYAML                        6.0
querystring-parser            1.2.4
regex                         2022.10.31
requests                      2.28.1
requests-auth-aws-sigv4       0.7
requests-oauthlib             1.3.1
responses                     0.22.0
rich                          12.6.0
rsa                           4.9
rstcheck                      6.1.1
rstcheck-core                 1.0.3
s3transfer                    0.6.0
scikit-learn                  1.1.3
scipy                         1.9.3
SecretStorage                 3.3.3
setuptools                    65.6.3
shap                          0.41.0
shellingham                   1.5.0
six                           1.16.0
slicer                        0.0.7
smmap                         5.0.0
sniffio                       1.3.0
snowballstemmer               2.2.0
Sphinx                        3.5.4
sphinx-autobuild              2021.3.14
sphinx-click                  4.3.0
sphinxcontrib-applehelp       1.0.2
sphinxcontrib-devhelp         1.0.2
sphinxcontrib-htmlhelp        2.0.0
sphinxcontrib-jsmath          1.0.1
sphinxcontrib-qthelp          1.0.3
sphinxcontrib-serializinghtml 1.1.5
SQLAlchemy                    1.4.44
sqlparse                      0.4.3
stack-data                    0.6.2
starlette                     0.22.0
starlette-exporter            0.14.0
tabulate                      0.9.0
tenacity                      8.1.0
threadpoolctl                 3.1.0
tokenize-rt                   5.0.0
tokenizers                    0.13.2
toml                          0.10.2
tomli                         2.0.1
tomlkit                       0.11.6
tornado                       6.2
tqdm                          4.64.1
traitlets                     5.6.0
transformers                  4.25.1
tritonclient                  2.28.0
typer                         0.7.0
types-docutils                0.19.1.1
types-toml                    0.10.8.1
typing_extensions             4.4.0
urllib3                       1.26.13
uvicorn                       0.20.0
uvloop                        0.17.0
virtualenv                    20.17.0
wcwidth                       0.2.5
websocket-client              1.4.2
Werkzeug                      2.2.2
wheel                         0.38.4
wrapt                         1.14.1
xgboost                       1.7.1
xmltodict                     0.13.0
yarl                          1.8.2
zipp                          3.11.0
zope.event                    4.5.0
zope.interface                5.5.2
```

Reply:
I think tensorlfow is required for pylint to infer `model` is callable.

Reply:
Wait this is a pytorch model. Is pytorch installed?

Reply:
Ahhh

Reply:
I run `dev/dev-env-setup.sh` to create venv first but `torch` does not installed even written in `requirements/extra-ml-requirements.txt`.

Reply:
Got it. Can we `# pylint: disable=not-callable`?